src/Selection.js

```diff
@@ -28,14 +43,17 @@ class Selection {
 
     this._listeners = Object.create(null);
 
-    this._mouseDown = this._mouseDown.bind(this)
-    this._mouseUp = this._mouseUp.bind(this)
+    this._initialEvent = this._initialEvent.bind(this)
+    this._closeSelector = this._closeSelector.bind(this)
```

Comment:
maybe `terminateEvent` or something more equivalent to `initialEvent` (and the old pattern of mouse down and up)?